recipe/meta.yaml

```diff
@@ -23,6 +23,8 @@ requirements:
     - cmake
     - pkg-config  # [unix]
     - python
+  always_include_files:  # [unix]
+    - lib/pkgconfig/jsoncpp.pc  # [unix]
```

Comment:
yes, but that file is not copied even with the always_include_files as the test fails